jsonparser/grafana.go

```diff
@@ -0,0 +1,219 @@
+package jsonparser
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/rudderlabs/rudder-go-kit/jsonrs"
+
+	"github.com/grafana/jsonparser"
+)
+
+// grafanaJSONParser is the implementation of JSONParser using jsonparser library
+type grafanaJSONParser struct{}
+
+// GetValue retrieves the value for a given key from JSON bytes using jsonparser
+func (p *grafanaJSONParser) GetValue(data []byte, keys ...string) (interface{}, error) {
+	if len(data) == 0 {
+		return nil, ErrEmptyJSON
+	}
+
+	// Handle empty key - return the entire JSON object
```

Comment:
Do we really want to handle an empty key like this? We could just return an error instead saying you're not supposed to call this with an empty key. If they want the whole thing they could just use json.Unmarshal then.